### PR TITLE
Draft PR for Adding support for Google T5 11b model to ROCm vLLM 

### PR DESCRIPTION
**Pull request by bmullick-amd**
Adding support for Google T5 11b model to ROCm vLLM

### CODE REVIEW

File: vllm/model_executor/models/t5.py [hunk 1]
```diff
@@ -0,0 +1,1017 @@
+# coding=utf-8
```
Comment (peiyolin) [inline vllm/model_executor/models/t5.py:1]: Unnecessary

Also, let's follow this file to add the source where we adapt this from: https://github.com/ROCm/vllm/blob/main/vllm/model_executor/models/falcon.py#L2

File: vllm/model_executor/models/t5.py [hunk 2]
```diff
@@ -0,0 +1,1017 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch T5 model."""
+
+
+import copy
+
+
+import math
```
Comment (peiyolin) [inline vllm/model_executor/models/t5.py:21]: There are a bunch of GitHub Actions, please fix them.

File: vllm/model_executor/models/t5.py (hunk 1 above)
Comment (bmullick-amd) [reply vllm/model_executor/models/t5.py]: added source

File: vllm/model_executor/models/t5.py (hunk 2 above)
Comment (bmullick-amd) [reply vllm/model_executor/models/t5.py]: fixed

**Comment by shajrawi** [comment]
Please create an upstream PR instead.

**Comment by bmullick-amd** [comment]
Adding T5 support to upstream vllm so that it can be adopted into rocm/vllm and closing this PR.

**Pull request closed without merging**